src/mobility/components/MobilityStats.tsx

```diff
@@ -0,0 +1,32 @@
+import {View} from 'react-native';
+import React from 'react';
+import {StyleSheet} from '@atb/theme';
+
+type MobilityStatsProps = {
+  first: JSX.Element;
```

Comment:
Rather than passing a JSX.Element here you could pass an object and rather use the <MobilityStat> component here.
```
type MobilityStat = { 
  svg?(props: SvgProps): JSX.Element;
  primaryStat: string | number;
  secondaryStat?: string | number;
  secondaryStatStyle?: TextStyle;
};

type MobilityStatsProps = {
    first: MobilityStat, <- or use props type exported from MobiltyStat
    second: MobilityStat,
```

Reply:
I don't think this would be a good idea because sometimes we want to pass something else such as `<PricingPlan>`, for example in `ScooterSheet.tsx` shown below

```
<MobilityStats
  first={
    <MobilityStat
      svg={Battery}
      primaryStat={vehicle.currentFuelPercent + '%'}
      secondaryStat={formatRange(vehicle.currentRangeMeters, language)}
    />
  }
  second={
    <PricingPlan
      operator={operatorName}
      plan={vehicle.pricingPlan}
      eligibleBenefit={
        operatorBenefit && isUserEligibleForBenefit
          ? operatorBenefit.id
          : undefined
      }
    />
  }
/>
```